lib/collections/schemas/tags.js

```diff
@@ -17,6 +18,14 @@ export const Tag = new SimpleSchema({
   slug: {
     type: String
   },
+  type: {
```

Comment:
type seems sort of vague? Just reading this I am not clear how I would use it?

Reply:
`type` has been (my)/the preferred name for anything that determines functionality, and should be a fairly standard behavior expectation across our schemas. 👍